src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -16,8 +16,19 @@
 var ReactFiberReconciler = require('ReactFiberReconciler');
 var ReactGenericBatching = require('ReactGenericBatching');
 var emptyObject = require('emptyObject');
+var ReactTypeOfWork = require('ReactTypeOfWork');
```

Comment:
In other words, after `npm run build` try cd-ing into `build/packages/react-test-renderer` and verify you can `require()` it without crashing.